ci/build.yml

```
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.21.5-bullseye
+    tag: 1.22.2-bullseye
```

Small thing but this is not the latest now… (https://hub.docker.com/_/golang)
```suggestion
    tag: 1.22.4-bullseye
```
(Also for the other files in this dir)